### DISCUSSION TITLE
How to query internal (bbdb) tables with JS Scripting in Automation?

### DISCUSSION THREAD

**Discussion by gerhardmpl** [Budibase/budibase#4514] [Help] [answered]
Can somebody please share a Javascript example on how to query internal (bbdb) tables (not bindings) with ```JS Scripting``` in Automation or direct me to the respective documentation or discussion?

**Comment by mjashanks** [comment, marked as answer]
It's not possible - the `Javascript` step only allows for data manipulations on structures from previous steps.

`Query Rows` is the only way.

**Comment by gerhardmpl** [comment]
Ok, thanks for clarifying. Can be closed since it is related to #4465.
